examples/webextension/README.md

```diff
@@ -0,0 +1,29 @@
+This example webextension creates a toolbar button and sends a ping to GA each time it's clicked.
+
+## installation
+
+- Edit `background.js`, set the value of the `TRACKING_ID` const to your GA tracking ID
```

Comment:
@ckprice Actually, Wil's metrics PR suggested leaving the author as owner throughout, but granting Mozilla access while it's in Test Pilot